tools/checkpoint_conversion/convert_gpt_neox_checkpoints.py

```diff
@@ -0,0 +1,200 @@
+        hf_wts[f"layers.{ilayer}.attention.query_key_value.weight"]
+        .numpy()
+        .T.reshape((cfg["hidden_dim"], cfg["num_heads"], -1))
+    )
+
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._self_attention_layer._qkv_dense.bias.assign(
+        hf_wts[f"layers.{ilayer}.attention.query_key_value.bias"].reshape(
+            (cfg["num_heads"], -1)
+        )
+    )
+
+    # Attention Dense
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._self_attention_layer._output_dense.kernel.assign(
+        hf_wts[f"layers.{ilayer}.attention.dense.weight"].numpy().T
+    )
+
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._self_attention_layer._output_dense.bias.assign(
+        hf_wts[f"layers.{ilayer}.attention.dense.bias"]
+    )
+
+    # LAYERNORM
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._input_layernorm.gamma.assign(
+        hf_wts[f"layers.{ilayer}.input_layernorm.weight"]
+    )
+
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._input_layernorm.beta.assign(
+        hf_wts[f"layers.{ilayer}.input_layernorm.bias"]
+    )
+
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._self_attention_layernorm.gamma.assign(
+        hf_wts[f"layers.{ilayer}.post_attention_layernorm.weight"]
+    )
+
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._self_attention_layernorm.beta.assign(
+        hf_wts[f"layers.{ilayer}.post_attention_layernorm.bias"]
+    )
+
+    # MLP
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._feedforward_intermediate_dense.kernel.assign(
+        hf_wts[f"layers.{ilayer}.mlp.dense_h_to_4h.weight"].numpy().T
+    )
+
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._feedforward_intermediate_dense.bias.assign(
+        hf_wts[f"layers.{ilayer}.mlp.dense_h_to_4h.bias"]
+    )
+
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._feedforward_output_dense.kernel.assign(
+        hf_wts[f"layers.{ilayer}.mlp.dense_4h_to_h.weight"].numpy().T
+    )
+
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._feedforward_output_dense.bias.assign(
+        hf_wts[f"layers.{ilayer}.mlp.dense_4h_to_h.bias"]
+    )
+
+    # Rotary Embedding
+    keras_model.get_layer(
+        f"transformer_layer_{ilayer}"
+    )._self_attention_layer.rotary_embedding.inverse_freq.assign(
+        hf_wts[f"layers.{ilayer}.attention.rotary_emb.inv_freq"]
+    )
+
+keras_model.get_layer("layer_norm").gamma.assign(
+    hf_wts["final_layer_norm.weight"]
+)
+
+keras_model.get_layer("layer_norm").beta.assign(hf_wts["final_layer_norm.bias"])
+
+hf_tokenizer = AutoTokenizer.from_pretrained(PRESET)
+sample_text = ["cricket is awesome, easily the best sport in the world!"]
+hf_inputs = hf_tokenizer(sample_text, return_tensors="pt")
+
+keras_inputs = {
+    "token_ids": tf.constant(
```

Comment:
can't we just convert hf_inputs directly? `tf.covert_to_tensor(torch_tensor)` should work

Reply:
cool !